en/getting_started/download_and_install.md

```diff
@@ -63,6 +65,10 @@ To install *QGroundControl* for Ubuntu Linux 16.04 LTS or later:
 
 > **Note** There are known [video steaming issues](../Support/troubleshooting_qgc.md#dual_vga) on Ubuntu 18.04 systems with dual adaptors.
 
+<span></span>
+> **Note** *QGroundControl* versions prior to 4.0 can run on Ubuntu 16.04.
```

Comment:
This isn't quite right. It says "versions prior to 4.0 can run on Ubuntu 16.04" which is correct. But then it says "to run later versions on Ubuntu 16.04 you can build yourself". The newest 4.0 version of QGC will run on Ubuntu 16.04 if you compile without video. We just don't provide those versions for download.

Reply:
Pah, it is 100% correct for 99.992 % of users, who are never going to build this with or without video. I'll re-write.

Reply:
Fixed. Merging now. If not precise enough then please update the note directly.